packages/block-editor/src/components/media-upload-progress/test/index.native.js

```diff
@@ -15,7 +15,7 @@ import {
 	MEDIA_UPLOAD_STATE_SUCCEEDED,
 	MEDIA_UPLOAD_STATE_FAILED,
 	MEDIA_UPLOAD_STATE_RESET,
-} from '../media-upload-progress';
+} from '../index';
```

Comment:
Minor suggestion, but it seems the most common way in other component tests is to import `'../` instead of `../index`

Reply:
Ohh I'll change it so its the same as the rest of the code 👍

Reply:
Fixed! https://github.com/WordPress/gutenberg/pull/17392/commits/c01681ce1973cfa530549b37af7ea5f4aacced05